src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -296,7 +229,7 @@ class ConsignmentRouteSpec extends AnyFlatSpec with Matchers with TestRequest wi
 
     val response: GraphqlQueryData = runTestQuery("query_somedata", validUserToken())
 
-    response.errors should have size 1
+    response.errors.size should be(1)
```

Comment:
What's the reason for this change? I'd have thought the original would give a better failure message.